cmd/util.go

```diff
@@ -27,14 +27,32 @@ var (
 
 var psFunc func(string) string
 var statFunc func(string) (os.FileInfo, error)
-var getBinariesFunc func(*viper.Viper) (map[string]string, error)
+var getBinariesFunc func(*viper.Viper, check.NodeType) (map[string]string, error)
 var TypeMap = map[string][]string{
 	"ca":         []string{"cafile", "defaultcafile"},
 	"kubeconfig": []string{"kubeconfig", "defaultkubeconfig"},
 	"service":    []string{"svc", "defaultsvc"},
 	"config":     []string{"confs", "defaultconf"},
 }
 
+var errMissingKubectlKubelet = fmt.Errorf(`Unable to find the programs kubectl or kubelet in the PATH.
```

Comment:
I think golint will complain about these as error messages (which by convention are not capitalized and aren't supposed to be full sentences). Perhaps there should be two things: 

- a short error message (e.g. "unable to find %s" in PATH")
- a more helpful human-readable text message which gets printed to stderr by a top-level function